lib/Installation/Partitioner/FormattingOptionsPage.pm

```diff
@@ -99,6 +100,9 @@ sub select_filesystem {
     elsif ($filesystem eq 'ext4') {
         send_key_until_needlematch(FILESYSTEM_EXT4, 'up');
     }
+    elsif ($filesystem eq 'xfs') {
+        send_key_until_needlematch(FILESYSTEM_XFS, 'up');
+    }
```

Comment:
We _could_ use a variable here (eg send_key_until_needlematch(FILESYSTEM_$variable) instead of that if..else table. We just need to put the lowercase in uppercase eg `my $variable = uc($filesystem)`. Same for select_partition_id above it.